apps/opencs/model/world/columns.cpp

```diff
@@ -62,7 +62,7 @@ namespace CSMWorld
             { ColumnId_StarterSpell, "Starter Spell" },
             { ColumnId_AlwaysSucceeds, "Always Succeeds" },
             { ColumnId_SleepForbidden, "Sleep Forbidden" },
-            { ColumnId_InteriorWater, "Interior Water" },
+            { ColumnId_Water, "Has Water" },
```

Comment:
Don't think that this is a good idea, because this really is only for interiors. Might confuse people when it shows up in an exterior cell.